packages/itwinui-variables/src/themes/dark-hc.scss

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@use 'sass:map';
+@use '../utils/hex' as *;
+
+$base-colors: (
+  // Background colors - default
+  background-1: hsl(200 12% 20%),
+  background-2: hsl(200 12% 15%),
+  background-3: hsl(200 12% 10%),
+  background-4: hsl(200 12% 25%),
+  background-5: hsl(200 12% 30%),
+  background-1-hover: hsl(0 0% 95%),
+  //
+  // Background colors - statuses subtle
+  background-subtle-primary: hsl(204 14% 44%),
```

Comment:
@FlyersPh9 We still need to fix these subtle background values in high contrast themes. Leaving this comment here as a reminder.

Reply:
Here are the values for dark high contrast subtle colors:
```scss
  background-subtle-primary: hsl(199 16% 24%),
  background-subtle-positive: hsl(167 17% 23%),
  background-subtle-warning: hsl(144 4% 24%),
  background-subtle-negative: hsl(210 5% 24%),
```